ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog.stories.ts

```diff
@@ -154,7 +154,7 @@ class DialogWithFormComponent {
           >Dialog with fudis-grid and scrollable content</fudis-heading
         >
         <fudis-dialog-content>
-          <fudis-grid [marginTop]="'md'" [marginBottom]="'md'">
+          <fudis-grid [classes]="['fudis-mt-md fudis-mb-md']">
```

Comment:
Seems like this margin-top does not affect at all, `mat-dialog__content` class is overriding it.